### PR TITLE
More typos

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
See https://github.com/Blosc/bloscpack/pull/117.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -18962,6 +18964,7 @@ laods->loads
 Laotion->Laotian, lotion,
 Laotions->Laotians, lotions,
 laout->layout
+larer->larger
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:18967]: Please also add `layer`  
`larer->larger, layer,`

Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:18967]: `Y` is not near `R` on (Qwerty) keyboards. What are the odds of typing `larer` instead of `layer`?

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:18967]: Please add both. TIA

Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt]: Done.

**Pull request merged**